configuration/order-updater/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/google/uuid"
+)
+
+var ORDER_IDS = []string{"orderId1", "orderId2"}
```

Comment:
Is this supposed to be here? Or inside the go directory?

Reply:
@sarsharma I see why you have written this. Issue is people trying out lets say js quickstart might not have the setup for go. 

If it is only meant to be used by the pipeline to simulate the keys update behind the scene then should we  clearly mention in readmes that `order-updater` is only meant for simulating the behavior and optional.

Reply:
@pravinpushkar agreed, but building binaries for all OS/ARCH combinations might not be feasible. Even if you are keeping this go code here, I would recommend moving order-updater to `utils` or something. As @pravinpushkar mentioned, (a) it can be optional, and (b) you can provide REDIS CLI commands to do the same thing.

Reply:
I think calling it out as optional and providing a redis-cli command or docker based redis cli command to update config values is needed here.
We can make use of the `order-updater` only for validations(in README comments) and in go quickstart alone.

Reply:
I really worry about end users needing Go to try the other languages.

Reply:
I removed `order-updater` and am now using `redis-cli` to add config items to the config store, and updating config items is optional for the user.